components/dropdown/demo/dropdown-fixed.html

```diff
@@ -0,0 +1,171 @@
+							<d2l-dropdown-content max-width="400" prefer-fixed-positioning>
+								<d2l-button-subtle id="add-content1">Add to Light</d2l-button-subtle>
+								<d2l-button-subtle id="add-content2">Add to Shadow</d2l-button-subtle>
+								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+									magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+									commodo consequat. </p>
+								<a href="http://www.desire2learn.com">D2L</a>
+							</d2l-dropdown-content>
+						</d2l-dropdown>
+						<br><br><br><br><br><br>
+					</div>
+				</div>
+				<script>
+					document.querySelector('#add-content1').addEventListener('click', e => {
+						const dropdown = e.target.parentNode.parentNode;
+						const newContent = document.createElement('p');
+						newContent.innerText = 'Trysail Sail ho Corsair red ensign hulk smartly boom jib rum gangway. Case shot Shiver me timbers gangplank crack Jennys tea cup ballast Blimey lee snow crow\'s nest rutters. Fluke jib scourge of the seven seas boatswain schooner gaff booty Jack Tar transom spirits.';
+						dropdown.parentNode.insertBefore(newContent, dropdown);
+					});
+					document.querySelector('#add-content2').addEventListener('click', () => {
+						const mutationDemo = document.querySelector('d2l-demo-mutation');
+						mutationDemo.count += 1;
+					});
+				</script>
+			</template>
+		</d2l-demo-snippet>
+
+		<h2>Dropdown (transform container)</h2>
+
+		<d2l-demo-snippet>
+			<template>
+
+				<div style="padding: 25px; transform: translate(0);">
+					<d2l-dropdown prefer-fixed-positioning>
+						<d2l-button class="d2l-dropdown-opener">Open it!</d2l-button>
+						<d2l-dropdown-content max-width="400" prefer-fixed-positioning>
+							<a href="https://youtu.be/9ze87zQFSak">Google</a>
+							<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+								magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+								commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
+								nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
+								anim id est laborum.</p>
+							<a href="http://www.desire2learn.com">D2L</a>
+						</d2l-dropdown-content>
+					</d2l-dropdown>
+				</div>
+
+			</template>
+		</d2l-demo-snippet>
+
+		<h2>Dropdown (in a dialog)</h2>
+
+		<d2l-demo-snippet>
+			<template>
+				<d2l-button id="openDialog1">Show Dialog</d2l-button>
+				<d2l-dialog id="dialog1" title-text="Dialog Title">
+					<div>
+						<p>Deadlights jack lad schooner scallywag dance the hempen jig carouser broadside cable strike colors. Bring a spring upon her cable holystone blow the man down spanker</p>
+						<d2l-dropdown prefer-fixed-positioning>
```

Comment:
Currently, the consumer would be required to provide `prefer-fixed-positioning` on both the opener and the content element. We can look at reducing this to just one by doing some DOM checking of the corresponding opener/content if need be.